main.rkt

```diff
@@ -8,6 +8,25 @@
 ;; and 'main.rkt' will just be the library's API.
 
 (provide
+ ;; --- constants
+ π
+ π/2
+ π/3
+ π/4
+ π/6
+ π/12
+ 2/π
+ 3π/4
+ e
+ 1/e
+ ;; --- helpers
+ square
```

Comment:
Bruh. `square` is in the library as [`sqr`](http://docs.racket-lang.org/reference/generic-numbers.html#%28def._%28%28lib._racket%2Fmath..rkt%29._sqr%29%29). LOL. I suppose this doesn't matter as the built in implementation is the same as ours. But I think that this indicates that it is more difficult to find the functions you need in the stdlib than it should be.